astropy/setup_package.py

```diff
@@ -1,5 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+import os
+import glob
+
 
 def get_package_data():
-    return {'astropy': ['astropy.cfg', 'CITATION']}
+
+    # Find all files in data/ sub-directories since this is a standard location
+    # for data files. We then need to adjust the paths to be relative to here
+    # (otherwise glob will be evaluated relative to setup.py)
+    data_files = glob.glob('**/data/**/*', recursive=True)
```

Comment:
I guess this is a small price to pay for overall simplicity. Does this ever need to be modified if new data files or sub-packages are added? Or when they are removed?

Reply:
@pllim - this will look a lot simpler once I move the definition of the data to setup.cfg - it will look something like:

```
[options.package_data]
astropy = **/data/**/*
```

(plus other files). We don't need to change anything if new ``data`` folders are added, or old ones are removed, or files are added/removed inside them.